compiler/il/ILProps.hpp

```diff
@@ -228,8 +228,7 @@ namespace ILProp2
       ZeroExtension                = 0x01000000,
       SignExtension                = 0x02000000,
       ByteSwap                     = 0x04000000,
-      // Available                 = 0x08000000,
-      // Available                 = 0x10000000,
+      BitPermute                   = 0x08000000,
```

Comment:
We seem to have deleted the `0x10000000` available line. We should leave that line in so others can use it. Otherwise a glossing eye may overlook that we jump from `0x08000000` to `0x20000000`

Also why do we need this flag exactly? Doesn't the IL imply this already?

Reply:
Yes, this was an accidental removal carried over from another set of changes.

The flag was added in an attempt to mirror ByteSwap's changes as a template for adding an opcode. Neither of the opcodes seem to make use of if. I'll remove `BitPermute`.